.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
+*   @opensearch-project/github
```

Comment:
is the team name `.github`? Oh well you can make a second PR if its busted

Reply:
@opensearch-project/github If you look at the team, that's the way it's formatted:  https://github.com/orgs/opensearch-project/teams/github  So I think it's right?

Reply:
Good point, somewhere there is a `.trim('.')` keeping the lights on at night :)